org.civicrm.bootstrap/scss/overrides/reset/_civicrm.scss

```diff
@@ -3,20 +3,20 @@
     #{$bootstrap-namespace} {
 
         h3 {
-            background-color: initial;
```

Comment:
A comment here saying that `initial` is not supported by IE and these are the initial values would be helpful, I think.